src/main/resources/testchipip/csrc/SimDRAM.cc

```diff
@@ -6,16 +6,17 @@
 #include "mm_dramsim2.h"
 
 int dramsim = -1;
-unsigned long loadmem_addr = 0;
 std::string ini_dir = "dramsim2_ini";
 std::string loadmem_file = "";
 
+// TODO FIX: This doesn't properly handle striped memory across multiple channels
```

Comment:
Why so?

Reply:
Currently, for multiple DRAM channels, this instantiates the full-memory size for each channel, even though  only a portion of it will be accessed due to the memory striping. 

This is a pre-existing issue, I'm just documenting it here for now.

Reply:
I added a comment to clarify